contracts/src/MyOperator.sol

```diff
@@ -1,60 +1,109 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
-import "@openzeppelin/contracts/utils/structs/EnumerableMap.sol";
-import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
 import "eigenlayer-contracts/src/contracts/core/DelegationManager.sol";
 import "eigenlayer-contracts/src/contracts/core/StrategyManager.sol";
 import "eigenlayer-contracts/src/contracts/interfaces/IDelegationManager.sol";
 import "eigenlayer-contracts/src/contracts/interfaces/IStrategy.sol";
 
 import {IEigenLayerContracts} from "./EigenLayerContracts.sol";
+import "./Vault.sol";
 
 contract MyOperator {
-    address public operatorAddress;
+    address public operator;
+
+    Vault vault;
```

Comment:
Why do we need this? 
Operator is could be strategy-agnostic in my understanding